### DISCUSSION TITLE
docker registry for Eclipse EDC components

### DISCUSSION THREAD

**Discussion by mnarvart** [eclipse-edc/Connector#3382] [Q&A] [answered]
Hello,
I was wondering if there is a public docker registry with connectors images for the tags done on connector repository or should people that need the image have to rebuild images on their side?

Thanks

**Comment by ndr-brt** [comment, marked as answer]
EDC is a framework, not a product, so it doesn't make sense to publish images because every dataspace have different needs and extensions.
The suggestion is to take a look at dataspace specific implementations like tractusx-edc:
 - [repository](https://github.com/eclipse-tractusx/tractusx-edc)
 - [docker hub](https://hub.docker.com/search?q=tractusx+edc)
